.pre-commit-config.yaml

```diff
@@ -31,5 +31,5 @@ repos:
   - id: doctoc
     files: |
       (?x)^(
-        CONTRIBUTING\.md
```

Comment:
Please remove remove the whole hook (`doctoc`) configuration since we're not using it anymore. We can reintroduce it later if we need it again.